classes/Bulk/Bulk.php

```diff
@@ -17,9 +17,11 @@ class Bulk {
 	 */
 	public function init() {
 		add_action( 'imagify_optimize_media', [ $this, 'optimize_media' ], 10, 3 );
-		add_action( 'imagify_convert_webp', [ $this, 'generate_webp_versions' ], 10, 2 );
+		add_action( 'imagify_convert_webp', [ $this, 'generate_next_gen_versions' ], 10, 2 );
+		add_action( 'imagify_convert_avif', [ $this, 'generate_next_gen_versions' ], 10, 2 );
```

Comment:
We could replace both with a new action `imagify_convert_next_gen`, and we decide what formats to generate in the callback

Reply:
Done